GenerateGraph.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import json
+from typing import AnyStr, List
+
+from DbInterface import Neo4jStream
+from TestGraphGenerator import export_database_neo4j, create_graph_map, export_database_json
+from TestGraphGenerator.DbTranformations import import_database_json
+from TestGraphGenerator.DbTranformations.DbLoader import delete_database_neo4j
+
+
+def load_names_data_set(path: AnyStr) -> List[AnyStr]:
+    """
+    Loads a list of names from a json file to use as properties in the test database.
+    :param path: the path to the file containing the names
+    :return: A list of names if the file is readable else returns none
+    """
+    with open(path) as data_file:
+        data_json = json.load(data_file)
+    return data_json['names']
+
+
+def create_arg_parser() -> argparse.ArgumentParser:
```

Comment:
This method seems out of place here.
This is a file for the graph generation, not the parser creation

Reply:
this file is a cli for the module for ease of use in case i want to use it from the terminal and not python code. this is the only fitting place for it as moving it to another class is not needed. 
the name comes as the name of the script otherwise it would be named graphgenerator cli but yknow